### PR TITLE
fix/UI Issue

### LINKED ISSUE #32: pronunciation button covering the text
Your Image Description 

OS: Android
Device name: Realme GT NEO2

### PR DESCRIPTION
**Pull request by ayato91**
Addition of physics can help to retrieve those covered words. 
fix #32

### CODE REVIEW

**Comment by ayato91** [comment]
@danger-ahead can you please check and share your review?

**Comment by danger-ahead** [comment]
Can you share a demo of the fix? Maybe a screenshot or recording?

**Comment by ayato91** [comment]
Sure, I will share those as well

**Comment by ayato91** [comment]
Preview added!
Scrolling Effect Animation

**Comment by danger-ahead** [comment]
> @danger-ahead Preview added! Scrolling Effect Animation [ Scrolling Effect Animation ](https://github.com/danger-ahead/flutter_dictionary/pull/images/Scroll_effect.gif) [ ](https://github.com/danger-ahead/flutter_dictionary/pull/images/Scroll_effect.gif)

Hey, Sorry for the confusion. I just wanted to see a demo. You don't need to add the file. Delete it, I have gone through the demo. And then, we'll be ready to merge.

File: images/Scroll_effect.gif
Comment (danger-ahead) [inline images/Scroll_effect.gif:1]: Need to remove this.

**Review by danger-ahead — CHANGES_REQUESTED** [review]
just 1 change, rest LGTM!

**Review by danger-ahead — APPROVED** [review]

**Pull request merged**